### PR TITLE
c2c-32: Added concepts with mappings to echographie

### PR DESCRIPTION
**Pull request by reagan-meant**
1. Changed `seat` to `breech` since its the accurate medical term
2.Changed fetal HR to numeric since it is a numeric value

### CODE REVIEW

File: configuration/concepts/echographie.csv [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-a7ad184c-b79a-476d-9ba2-ef980375da28,,,Fœtus Number,Numero du Fœtus,Fœtus #,№ du Fœtus,,,Misc,Numeric,,,,,,,,,,,
-7ce5a9fe-d9e0-427e-a080-ce8a9c673822,,,Fetal heart rate,Rythme cardiaque foetus,Fetal heart rate,Rythme cardiaque foetus,,,Misc,Text,,,,,,,,,,,
-c4526510-3f10-11e4-adec-0800271c1b75,,,Cephalic,Céphalique,Cephalic,Céphalique,,,Misc,N/A,,,,,,,,,,,
-a0dc5727-c33e-479b-b70e-fb0eb7e45853,,,Seat,Siège,Seat,Siège,,,Misc,N/A,,,,,,,,,,,
-5bb2ea3b-a736-4533-bd5e-a1e9d835f721,,,Transversal,Transversale,Transversal,Transversale,,,Misc,N/A,,,,,,,,,,,
+a7ad184c-b79a-476d-9ba2-ef980375da28,,CIEL:165387,Fœtus Number,Numero du Fœtus,Fœtus #,№ du Fœtus,,,Misc,Numeric,,,,,,,,,,,
+7ce5a9fe-d9e0-427e-a080-ce8a9c673822,,CIEL:1440,Fetal heart rate,Rythme cardiaque foetus,Fetal heart rate,Rythme cardiaque foetus,,,Misc,Numeric,,,,FALSE,beats/min,120,160,40,250,,
+c4526510-3f10-11e4-adec-0800271c1b75,,CIEL:160091,Cephalic,Céphalique,Cephalic,Céphalique,,,Misc,N/A,,,,,,,,,,,
+a0dc5727-c33e-479b-b70e-fb0eb7e45853,,CIEL:146922,Breech,Culasse,Breech,Culasse,,,Misc,N/A,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/echographie.csv:5]: "Siège" in FR

File: configuration/concepts/echographie.csv [hunk 2]
```diff
@@ -1,11 +1,11 @@
 uuid,Void/Retire,Same as mappings,Fully specified name:en,Fully specified name:fr,Short name:en,Short name:fr,Description:en,Description:fr,Data class,Data type,Complex data handler,Answers,Members,Allow decimals,Units,Normal low,Normal high,Absolute low,Absolute high,_version:1,_order:1100
-a7ad184c-b79a-476d-9ba2-ef980375da28,,,Fœtus Number,Numero du Fœtus,Fœtus #,№ du Fœtus,,,Misc,Numeric,,,,,,,,,,,
-7ce5a9fe-d9e0-427e-a080-ce8a9c673822,,,Fetal heart rate,Rythme cardiaque foetus,Fetal heart rate,Rythme cardiaque foetus,,,Misc,Text,,,,,,,,,,,
-c4526510-3f10-11e4-adec-0800271c1b75,,,Cephalic,Céphalique,Cephalic,Céphalique,,,Misc,N/A,,,,,,,,,,,
-a0dc5727-c33e-479b-b70e-fb0eb7e45853,,,Seat,Siège,Seat,Siège,,,Misc,N/A,,,,,,,,,,,
-5bb2ea3b-a736-4533-bd5e-a1e9d835f721,,,Transversal,Transversale,Transversal,Transversale,,,Misc,N/A,,,,,,,,,,,
+a7ad184c-b79a-476d-9ba2-ef980375da28,,CIEL:165387,Fœtus Number,Numero du Fœtus,Fœtus #,№ du Fœtus,,,Misc,Numeric,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/echographie.csv:2]: "Numéro"

File: configuration/concepts/echographie.csv [hunk 3]
```diff
@@ -1,11 +1,11 @@
...
+7ce5a9fe-d9e0-427e-a080-ce8a9c673822,,CIEL:1440,Fetal heart rate,Rythme cardiaque foetus,Fetal heart rate,Rythme cardiaque foetus,,,Misc,Numeric,,,,FALSE,beats/min,120,160,40,250,,
```
Comment (rbuisson) [inline configuration/concepts/echographie.csv:3]: "Rythme cardiaque fœtal"

File: configuration/concepts/echographie.csv [hunk 4]
```diff
@@ -1,11 +1,11 @@
...
+c4526510-3f10-11e4-adec-0800271c1b75,,CIEL:160091,Cephalic,Céphalique,Cephalic,Céphalique,,,Misc,N/A,,,,,,,,,,,
+a0dc5727-c33e-479b-b70e-fb0eb7e45853,,CIEL:146922,Breech,Culasse,Breech,Culasse,,,Misc,N/A,,,,,,,,,,,
+5bb2ea3b-a736-4533-bd5e-a1e9d835f721,,CIEL:112259,Transverse,Transversale,Transverse,Transversale,,,Misc,N/A,,,,,,,,,,,
 cbafba29-b62a-4675-910a-a6bb161de256,,,Variable,Variable,Variable,Variable,,,Misc,N/A,,,,,,,,,,,
-d47e2e11-355e-4c62-86a7-8ebe45fb4292,,,Presentation,Presentation,Presentation,Presentation,,,Misc,Coded,,Cephalic;Seat;Transversal;Variable,,,,,,,,,
+d47e2e11-355e-4c62-86a7-8ebe45fb4292,,CIEL:160090,Presentation,Presentation,Presentation,Presentation,,,Misc,Coded,,Cephalic;Breech;Transverse;Variable,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/echographie.csv:8]: "Présentation"

**Review by rbuisson — CHANGES_REQUESTED** [review]
See comments

File: configuration/concepts/echographie.csv (hunk 1 above)
Comment (reagan-meant) [reply configuration/concepts/echographie.csv]: done

File: configuration/concepts/echographie.csv (hunk 2 above)
Comment (reagan-meant) [reply configuration/concepts/echographie.csv]: done

File: configuration/concepts/echographie.csv (hunk 3 above)
Comment (reagan-meant) [reply configuration/concepts/echographie.csv]: done

File: configuration/concepts/echographie.csv (hunk 4 above)
Comment (reagan-meant) [reply configuration/concepts/echographie.csv]: done

**Pull request merged**